appknox/cli.py

```diff
@@ -462,5 +462,29 @@ def summary_csv(ctx, report_id, output):
         sys.exit(1)
 
 
+@download_report.command("summary-excel")
+@click.argument("report_id", required=True, type=int)
+@click.option(
+    "-o",
+    "--output",
+    required=True,
+    type=click.Path(writable=True),
```

Comment:
Please change the output option type to Click.Path for summary-csv as well and test it.

Reply:
It's already there in summary-csv command. My bad